pyproject.toml

```diff
@@ -91,6 +91,7 @@ dependencies = [
     "openai",
     "panel @ git+https://github.com/holoviz/panel.git@main",
     "pypdf",
+    "tabulate",
```

Comment:
Is this needed? There's already quite a bit of requirements.

Reply:
Yes. Otherwise this example errors on Windows. Its needed when you create the Pandas Dataframe Agent.

Reply:
We can make it required on Windows only?

Reply:
Ok, I just didn't see it used explicitly.

Eventually, I think we may need to split installs by `panel-chat-examples[openai]` and `panel-chat-examples[langchain]`

Reply:
`tabulate` is also needed on linux. I had it installed without knowing it. I tried uninstalling and when you send the dataframe you get. So its needed.

![image](https://github.com/holoviz-topics/panel-chat-examples/assets/42288570/db49a003-ec58-481f-9b36-b3beeb8994c1)